demisto_sdk/commands/format/update_generic_yml.py

```diff
@@ -81,6 +84,15 @@ def update_id_to_equal_name(self) -> None:
             if is_uuid(self.id_and_version_location['id']):
                 updated_integration_id[self.id_and_version_location['id']] = self.data['name']
             self.id_and_version_location['id'] = self.data['name']
+        else:
+            if self.verbose:
+                click.echo('It is a modified file, keeping the old ID')
```

Comment:
I don't see the purpose of this message as we're already printing in case of a modification.